### PR TITLE
Fix warning, see issue #85

### PR DESCRIPTION
**Pull request by vtorri**

### CODE REVIEW

**Comment by vtorri** [comment]
any news on this simple PR ?

File: graph.c [hunk 1]
```diff
@@ -119,7 +119,7 @@ nodepath(struct node *n, bool escape)
 	escape = false;
 	nquote = 0;
 	for (s = n->path->s; *s; ++s) {
-		if (!isalnum(*s) && !strchr("_+-./", *s))
+		if (!isalnum((int)(*s)) && !strchr("_+-./", *s))
```
Comment (E5ten) [inline graph.c:122]: The parens around `*s` aren't necessary

Comment (vtorri) [reply graph.c:122]: and ?

Comment (E5ten) [reply graph.c:122]: And they should be removed, because redundant parens next to casts aren't used elsewhere in the code.

Comment (vtorri) [reply graph.c:122]: let's wait for @michaelforney 's answer

Comment (michaelforney) [inline graph.c:122]: The warning is legitimate, but this is not a correct fix. The C standard says that if the value is not representable as `unsigned char`, we get undefined behavior. If `char` is signed, and has a negative value (for example, one of the code units of some multibyte UTF-8 sequence), then converting to `int` still retains the negative value.

Instead, we want to interpret the the pointed-to `char` as `unsigned char`:

```C
isalnum(*(unsigned char *)s);
```

Comment (michaelforney) [reply graph.c:122]: Yes, please remove the unnecessary parentheses to be consistent with the rest of the code.

**Comment by vtorri** [comment]
@michaelforney should be good now

**Comment by michaelforney** [comment]
Thanks! Applied.

**Pull request closed without merging**